syntax = "proto3";

package bluetooth.hci;

import "google/protobuf/empty.proto";

service ControllerFacade {
  rpc GetMacAddress(google.protobuf.Empty) returns (AddressMsg) {}
  rpc WriteLocalName(NameMsg) returns (google.protobuf.Empty) {}
  rpc GetLocalName(google.protobuf.Empty) returns (NameMsg) {}
}

message AddressMsg {
  bytes address = 1;
}

message NameMsg {
  bytes name = 1;
}
